### PR TITLE
Show the delta value of different metrics when comparing runs in experiments tracking. 

### PR DESCRIPTION
**Pull request by rashidakanchwala**
Signed-off-by: Rashida Kanchwala <rashida_kanchwala@mckinsey.com>

## Description

Resolves - #1184 

## Development notes

This is from the design 

Screenshot 2023-02-15 at 11 39 36

This is how it looks

Screenshot 2023-02-15 at 11 44 22

## Checklist

- [x] Read the [contributing](/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [x] Added new entries to the `RELEASE.md` file
- [ ] Added tests to cover my changes

### CODE REVIEW

File: src/components/experiment-tracking/run-dataset/run-dataset.js [hunk 1]
```diff
@@ -226,8 +239,9 @@ function buildDatasetDataMarkup(
   const isPlotlyDataset = getShortType(datasetType) === 'plotly';
   const isImageDataset = getShortType(datasetType) === 'image';
   const isJSONTrackingDataset = getShortType(datasetType) === 'JSONTracking';
-  const isMetricsTrackingDataset = getShortType(datasetType) === 'metricsTracking';
-  const isTrackingDataset = isJSONTrackingDataset || isMetricsTrackingDataset
+  const isMetricsTrackingDataset =
+    getShortType(datasetType) === 'metricsTracking';
+  const isTrackingDataset = isJSONTrackingDataset || isMetricsTrackingDataset;
```
Comment (tynandebold) [inline src/components/experiment-tracking/run-dataset/run-dataset.js:244]: What are these changes?

**Review by tynandebold — APPROVED** [review]
Looks great! Approving with one question.

Please add a line to the releases notes before merging :)

File: src/components/experiment-tracking/run-dataset/run-dataset.js (hunk 1 above)
Comment (rashidakanchwala) [reply src/components/experiment-tracking/run-dataset/run-dataset.js:244]: i think just pretify. we did this change in an old PR - the json view release.

File: src/components/experiment-tracking/run-dataset/run-dataset.js [hunk 2]
```diff
@@ -24,12 +24,26 @@ const determinePinIcon = (data, pinValue, pinnedRun) => {
...
 };
 
+const determinePinDelta = (data, pinValue, pinnedRun) => {
+  if (
+    data.runId !== pinnedRun &&
+    typeof data.value === 'number' &&
+    data.value !== pinValue
+  ) {
+    const delta = data.value - pinValue;
+    const deltaPercentage = Math.round((delta / pinValue) * 100);
```
Comment (tynandebold) [inline src/components/experiment-tracking/run-dataset/run-dataset.js:34]: I think we need to watch out for when there are negative metric values.

image

This percentage figure here should be positive, right? Going from -3.6 to 1.4 is a positive increase.

Comment (rashidakanchwala) [reply src/components/experiment-tracking/run-dataset/run-dataset.js:34]: good catch :)

**Review by tynandebold — APPROVED** [review]
Looks good!

**Review by Huongg — APPROVED** [review]
Thanks @rashidakanchwala , everything is working fine for me too 👍

**Pull request merged**